packages/dds/map/package.json

```diff
@@ -106,8 +107,12 @@
 	"typeValidation": {
 		"version": "2.0.0-internal.3.2.0",
 		"previousVersionStyle": "~previousMinor",
-		"baselineRange": ">=2.0.0-internal.3.1.0 <2.0.0-internal.3.2.0",
-		"baselineVersion": "2.0.0-internal.3.1.0",
-		"broken": {}
+		"baselineRange": ">=2.0.0-internal.3.0.0 <2.0.0-internal.3.1.0",
```

Comment:
these fields shouldn't be updated. looks like a merge conflict with a more recent release branch. "broken" should be the only changed key.